packages/create/src/generators/wc-lit-element-ts/templates/_tsconfig.json

```diff
@@ -0,0 +1,15 @@
+  {
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "esnext",
+    "moduleResolution": "node",
+    "noEmitOnError": true,
+    "lib": ["es2017", "dom"],
+    "strict": true,
+    "esModuleInterop": true,
```

Comment:
This should be false, it's for commonjs modules but we don't support that anyway.